workloads/network-perf/README.md

```diff
@@ -17,35 +17,52 @@ $ ./run_<test-name>_network_test_fromgit.sh
 ## Environment variables
 
 ### ES_USER
-Default: `` 
+Default: ``    
 Username for elasticsearch instance
 
 ### ES_PASSWORD
-Default: `` 
+Default: ``     
 Password for elasticsearch instance
 
 ### ES_SERVER
-Default: `milton.aws.com`  
+Default: `milton.aws.com`    
 Elasticsearch server to index the results of the current run
 
 ### ES_PORT
-Default: ``  
+Default: ``    
 Port number for elasticsearch server
 
 ### METADATA_COLLECTION
 Default: `true`   
 Enable/Disable collection of metadata
 
 ### COMPARE
-Default: `false`   
+Default: `false`        
 Enable/Disable the ability to compare two uperf runs. If set to `true`, the next set of environment variables pertaining to the type of test are required
 
+### COMPARE_WITH_GOLD
+Default: ``     
+If COMPARE is set to true and COMPARE_WITH_GOLD is set to true then the current run will be compared against our gold-index
+Note: Make sure that elasticsearch baseline uuid (example: BASELINE_POD_1P_UUID, BASELINE_POD_2P_UUID ...) vars are not set or else it will override the uuids
+
+### GOLD_SDN
+Default: `you current cluster's sdn`   
```

Comment:
Lets just default this to openshiftsdn for now

Reply:
done